<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDroneDashboard</class>
 <widget class="QWidget" name="QDroneDashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>160</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QRadioButton" name="rbConnectionState">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>142</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>88</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>connected</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QRadioButton" name="rbAlertState">
     <property name="minimumSize">
      <size>
       <width>88</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>88</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>alert</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLCDNumber" name="lcdLinkQuality">
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>signal</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLCDNumber" name="lcdCharge">
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>% battery</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLCDNumber" name="lcdSpeedCMS">
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>cm/s</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLCDNumber" name="lcdTurnVal">
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>turn-val</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLCDNumber" name="lcdSpeedVal">
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="numDigits">
      <number>5</number>
     </property>
     <property name="segmentStyle">
      <enum>QLCDNumber::Flat</enum>
     </property>
     <property name="value" stdset="0">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>speed-val</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="lePosture">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>71</width>
       <height>23</height>
      </size>
     </property>
     <property name="text">
      <string>posture</string>
     </property>
     <property name="buddy">
      <cstring>lePosture</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
